pkg/apis/batch/v2alpha1/defaults_test.go

```diff
@@ -20,179 +20,58 @@ import (
 	"reflect"
 	"testing"
 
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/kubernetes/pkg/api"
 	_ "k8s.io/kubernetes/pkg/api/install"
-	"k8s.io/kubernetes/pkg/api/v1"
 	_ "k8s.io/kubernetes/pkg/apis/batch/install"
 	. "k8s.io/kubernetes/pkg/apis/batch/v2alpha1"
 )
 
-func TestSetDefaultJob(t *testing.T) {
-	defaultLabels := map[string]string{"default": "default"}
+func TestSetDefaultCronJob(t *testing.T) {
```

Comment:
This is adding new test for CronJob, not removing Job from v2alpha1. This is okay for now, but in the future, I'd recommend separating this as its own commit or PR. This way, it'll be easier to review and manage (e.g. revert changes).